include/picongpu/particles/atomicPhysics/rateCalculation/BoundFreeFieldTransitionRates.hpp

```diff
@@ -71,54 +141,100 @@ namespace picongpu::particles::atomicPhysics::rateCalculation
+         * @tparam T_AtomicStateDataBox instantiated type of dataBox
+         * @tparam T_BoundFreeTransitionDataBox instantiated type of dataBox
+         *
+         * @param maxEFieldNorm maximum E-field vector norm, in sim.units.eField()
+         * @param minEFieldNorm minimum E-field vector norm, in sim.units.eField()
+         * @param ionizationPotentialDepression, in eV
+         * @param transitionCollectionIndex index of transition in boundBoundTransitionDataBox
+         * @param atomicStateDataBox access to atomic state property data
+         * @param boundBoundTransitionDataBox access to bound-bound transition data
+         *
+         * @return unit: 1/picongpu::sim.unit.time()
+         */
+        template<
+            typename T_EFieldType,
+            typename T_ChargeStateDataBox,
+            typename T_AtomicStateDataBox,
+            typename T_BoundFreeTransitionDataBox>
+        HDINLINE static float_X maximumRateADKFieldIonization(
+            T_EFieldType const maxEFieldNorm,
+            T_EFieldType const minEFieldNorm,
+            float_X const ionizationPotentialDepression,
+            uint32_t const transitionCollectionIndex,
+            T_ChargeStateDataBox const chargeStateDataBox,
+            T_AtomicStateDataBox const atomicStateDataBox,
+            T_BoundFreeTransitionDataBox const boundFreeTransitionDataBox)
+        {
+            if(maxEFieldNorm == 0._X)
+                return 0._X;
 
-            // factor from averaging over one laser cycle with LINEAR polarization
-            if constexpr(
-                u32(T_ADKLaserPolarization) == u32(atomicPhysics::enums::ADKLaserPolarization::linearPolarization))
-                rateADK_AU *= math::sqrt(3._X * nEffCubed * eFieldNorm_AU / (pi * screenedChargeCubed));
+            float_X const Z = screenedCharge(
+                transitionCollectionIndex,
+                chargeStateDataBox,
+                atomicStateDataBox,
+                boundFreeTransitionDataBox);
 
-            /* A * 1/sim.atomicUnit.time() = A * 1/sim.atomicUnit.time() * sim.unit.time() / sim.unit.time()
-             *   = A * [sim.unit.time()/sim.atomicUnit.time()] * 1/sim.unit.time()
-             *   = (A * timeConversion) * 1/sim.unit.time()
-             *   = B * 1/sim.unit.time() */
-            constexpr float_X timeConversion = picongpu::sim.unit.time() / picongpu::sim.atomicUnit.time();
+            // Hartree
+            float_X const ionizationEnergy = picongpu::sim.si.conv().eV2auEnergy(DeltaEnergyTransition::get(
+                transitionCollectionIndex,
+                atomicStateDataBox,
+                boundFreeTransitionDataBox,
+                ionizationPotentialDepression,
+                chargeStateDataBox));
 
-            // 1/ sim.unit.time()
-            return rateADK_AU * timeConversion;
+            float_X const nEff = effectivePrincipalQuantumNumber(Z, ionizationEnergy);
```

Comment:
Offline discussion: Extract method for preparing all of this to remove code duplication with the above method.

Reply:
done